websites/api.py

```diff
@@ -224,3 +228,18 @@ def detect_mime_type(uploaded_file: UploadedFile) -> str:
     magic = Magic(mime=True)
     chunk = next(uploaded_file.chunks(chunk_size=2048))
     return magic.from_buffer(chunk)
+
+
+def reset_publishing_fields(website_name: str):
+    """Reset all publishing fields to allow a fresh publish request"""
+    now = now_in_utc()
+    Website.objects.filter(name=website_name).update(
+        has_unpublished_live=True,
+        has_unpublished_draft=True,
+        live_publish_status=PUBLISH_STATUS_NOT_STARTED,
```

Comment:
At the moment the frontend assumes that `PUBLISH_STATUS_NOT_STARTED` is one of the processing states where  it should keep polling until the status changes. It may be better to set this and `draft_publish_status` to `None` if we want the status to remain as is until the next time the user starts a publish